### PR TITLE
Improved admin_scan

### PR DESCRIPTION
**Pull request by aman566**
The output of admin_scan is not coming correctly.

 Steps to reproduce:
1. `python3 nettacker.py -m admin_scan -i https://pintasuper.com`

The tools give false positives.

It might be due to the time frame between two requests So I added a small time between two requests. The results come appropriate after that.

#### Checklist
- [X] I have followed the [Contributor Guidelines](https://github.com/zdresearch/OWASP-Nettacker/wiki/Developers#contribution-guidelines).
- [X] I have added the relevant documentation.
- [X] My branch is up-to-date with the Upstream master branch.

#### Your development environment
- OS: `Linux`
- OS Version: `Ubuntu`
- Python Version: `3.6.9`

### CODE REVIEW

File: lib/scan/admin/engine.py [hunk 1]
```diff
@@ -90,16 +91,23 @@ def check(target, user_agent, timeout_sec, log_in_file, language, time_sleep, th
...
-                        break
+        if r.status_code is 200:
+            for dlmsg in directory_listing_msgs:
+                if dlmsg in content:
+                    info(messages(language, "directory_listing").format(target), log_in_file, "a"
+                                        ,{'HOST': target_to_host(target), 'USERNAME': '', 'PASSWORD': '',
+                                        'PORT': "", 'TYPE': 'admin_scan',
+                                        'DESCRIPTION': messages(language, "directoy_listing").format(target), 'TIME': now(),
+                                        'CATEGORY': "scan", 'SCAN_ID': scan_id, 'SCAN_CMD': scan_cmd}, language, thread_tmp_filename)
+                    __log_into_file(log_in_file, 'a', data, language)
```
Comment (securestep9) [inline lib/scan/admin/engine.py:102]: there is an old bug here - pylint: lib/scan/admin/engine.py:257:50: E0602: Undefined variable 'data' (undefined-variable)

File: lib/scan/admin/engine.py [hunk 2]
```diff
@@ -243,7 +252,7 @@ def start(target, users, passwds, ports, timeout_sec, thread_number, num, total,
                 info(messages(language, "directory_file_404").format(
                     target, "default_port"), log_in_file, "a",
                     {'HOST': target_to_host(target), 'USERNAME': '', 'PASSWORD': '', 'PORT': '', 'TYPE': 'admin_scan',
-                     'DESCRIPTION': messages(language, "direcroty_file_404").format(target, "default_port"), 'TIME': now(), 'CATEGORY': "scan", 'SCAN_ID': scan_id,
+                     'DESCRIPTION': messages(language, "directory_file_404").format(target, "default_port"), 'TIME': now(), 'CATEGORY': "scan", 'SCAN_ID': scan_id,
                      'SCAN_CMD': scan_cmd}, language, thread_tmp_filename)
                 __log_into_file(log_in_file, 'a', data, language)
```
Comment (securestep9) [inline lib/scan/admin/engine.py:257]: old bug here which needs fixing: pylint: lib/scan/admin/engine.py:257:50: E0602: Undefined variable 'data' (undefined-variable)

File: lib/scan/admin/engine.py [hunk 3]
```diff
@@ -90,16 +91,23 @@ def check(target, user_agent, timeout_sec, log_in_file, language, time_sleep, th
...
-                                           ,{'HOST': target_to_host(target), 'USERNAME': '', 'PASSWORD': '',
-                                           'PORT': "", 'TYPE': 'admin_scan',
-                                           'DESCRIPTION': messages(language, "directoy_listing").format(target), 'TIME': now(),
-                                           'CATEGORY': "scan", 'SCAN_ID': scan_id, 'SCAN_CMD': scan_cmd}, language, thread_tmp_filename)
-                        __log_into_file(log_in_file, 'a', data, language)
-                        break
+        if r.status_code is 200:
+            for dlmsg in directory_listing_msgs:
+                if dlmsg in content:
+                    info(messages(language, "directory_listing").format(target), log_in_file, "a"
```
Comment (securestep9) [inline lib/scan/admin/engine.py:97]: OLD bug here as "directory_listing"  is actually "directoy_listing" with a typo everywhere

File: lib/scan/admin/engine.py [hunk 4]
```diff
@@ -90,16 +91,23 @@ def check(target, user_agent, timeout_sec, log_in_file, language, time_sleep, th
...
-                for dlmsg in directory_listing_msgs:
-                    if dlmsg in content:
-                        info(messages(language, "directory_listing").format(target), log_in_file, "a"
-                                           ,{'HOST': target_to_host(target), 'USERNAME': '', 'PASSWORD': '',
-                                           'PORT': "", 'TYPE': 'admin_scan',
-                                           'DESCRIPTION': messages(language, "directoy_listing").format(target), 'TIME': now(),
-                                           'CATEGORY': "scan", 'SCAN_ID': scan_id, 'SCAN_CMD': scan_cmd}, language, thread_tmp_filename)
-                        __log_into_file(log_in_file, 'a', data, language)
-                        break
+        if r.status_code is 200:
```
Comment (securestep9) [inline lib/scan/admin/engine.py:94]: "is" needs to be "==" here and in other places:
flake8:
lib/scan/admin/engine.py:84:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:94:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:202:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:240:35: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:245:20: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:250:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:251:16: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)

**Comment by aman566** [comment]
I have fixed the bugs which were suggested.

**Pull request merged**